source/common/stats/allocator_impl.h

```diff
@@ -46,8 +46,13 @@ class AllocatorImpl : public Allocator {
     bool operator()(const Metric* a, const Metric* b) const {
       return a->statName() == b->statName();
     }
-    bool operator()(StatName a, const Metric* b) const { return a == b->statName(); }
     bool operator()(const Metric* a, StatName b) const { return a->statName() == b; }
+    // This comparison pattern does not currently occur due to the implementation of
```

Comment:
WDYT about just deleting this for now and we can fix the compiler error later?

Reply:
+1, or just leave it

Reply:
I don't have a strong opinion; but are you asking to delete the entire comment? Or just the commented out method?

Reply:
I would just delete all of it personally.